### ISSUE TITLE
Changing anything (temperature, heat, off) from homekit will result in a HAP status 400 error

### ISSUE THREAD

**Issue by bramsmulders** [homebridge-plugins/homebridge-resideo#505]
**Describe The Bug:**

Changing anything (temperature, heat, off) from homekit will result in a HAP status 400 error.

**To Reproduce:**

Add thermostat to homebridge -> change temperature from home app.

**Expected behavior:**

No errors and a working thermostat from homekit.

**Logs:**

```
[09/06/2021, 15:16:04] [HoneywellHome] Sending request for Thermostat Thermostat to Honeywell API. mode: Off coolSetpoint: 10 heatSetpoint: 20 thermostatSetpointStatus: PermanentHold
[09/06/2021, 15:16:05] [HoneywellHome] "Request failed with status code 400"
(node:16930) UnhandledPromiseRejectionWarning: Error: HAP Status Error: -70408
    at new HapStatusError (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/hapStatusError.ts:15:5)
    at Thermostats.apiError (/usr/local/lib/node_modules/homebridge-honeywell-home/src/devices/thermostats.ts:634:11)
    at /usr/local/lib/node_modules/homebridge-honeywell-home/src/devices/thermostats.ts:261:16
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:16930) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:16930) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Homebridge Config:**

```json
{
            "name": "HoneywellHome",
            "credentials": {
                "consumerKey": "",
                "consumerSecret": "",
                "refreshToken": "",
                "notice": "Keep your tokens a secret!"
            },
            "options": {
                "thermostat": {
                    "thermostatSetpointStatus": "PermanentHold"
                },
                "roompriority": {
                    "priorityType": "PickARoom"
                }
            },
            "platform": "HoneywellHome"
        }
```

**Device & Model:**

T6 Wired

# **Environment:**

- **Node.js Version**: v14.17.0
- **NPM Version**: v6.14.13
- **Homebridge Version**: 
- **Homebridge Honeywell Home Plugin Version**: v9.4.2
- **Homebridge Config UI X Plugin Version**: v4.41.1
- **Operating System**: Raspbian

**Comment by jakeflavin** [comment]
I’m also seeing this same error after updating the plugin.

**Comment by donavanbecker** [comment]
Are any other versions of the plugin working?

**Comment by jakeflavin** [comment]
Sorry. I haven’t had the bandwidth to look into this until today. 

No other versions of the plug in worked for me. I even downgraded the plug-in, homebridge, and node to the version they were before I saw the issue. Since then I’ve upgraded all of them to the most recent versions and still saw this error. 

I unlinked and re-linked my Honeywell account and deleted the Honeywell cache in home bridge and that seemed to fix it.
